Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -7617,6 +9033,23 @@ def generate_simple_context_data_command_results(
     return command_results
 
 
+def validate_argument(args: dict[str, Any], key_: str, values: List[str] | None = None):
```

Comment:
As I mentiond in the ArgumentValues class, why do we need to validate predefined args?